src/taipy/gui/gui.py

```diff
@@ -1138,7 +1138,10 @@ def _call_user_callback(
                 self.__set_client_id_in_context(state_id)
                 if module_context is not None:
                     self._set_locals_context(module_context)
-                return self._call_function_with_state(user_callback, args)
+                callback_result = self._call_function_with_state(user_callback, args)
```

Comment:
Wasn't a 'finally' statement ever more clear?

Reply:
and it would allow for exception management